python/ray/autoscaler/aws/cloudwatch/example-cloudwatch-agent-config.json

```diff
@@ -8,6 +8,74 @@
       "port":31000
    },
    "logs":{
+      "metrics_collected": {
+        "prometheus": {
+          "log_group_name": "{cluster_name}-Prometheus",
```

Comment:
If a user starts and tears down ray cluster 1, and then restarts cluster 2 with the `cluster_name`, will the metrics of cluster 1 & 2 overlap?

Reply:
It will be under same {cluster_name}-prometheus namespace if same cluster name is given, but the metrics will not be overwritten instance ip address has changed. Metrics for both cluster 1 and 2 are shown on CloudWatch as different metrics.

Reply:
> If a user starts and tears down ray cluster 1, and then restarts cluster 2 with the `cluster_name`, will the metrics of cluster 1 & 2 overlap?

Metrics for both cluster 1 and 2 will be under same {cluster_name}-prometheus namespace if same cluster name is given, but the metrics will not be overwritten instance ip address has changed. Metrics for both cluster 1 and 2 are shown on CloudWatch as different metrics.